modules/services/rpc_service.lua

```diff
@@ -0,0 +1,194 @@
+require('atf.util')
+
+local functionId = require('function_id')
+local json = require('json')
+local constants = require('protocol_handler/ford_protocol_constants')
+local events = require('events')
+local Event = events.Event
+local expectations = require('expectations')
+local load_schema = require('load_schema')
+local mob_schema = load_schema.mob_schema
+
+local Expectation = expectations.Expectation
+local Event = events.Event
+local SUCCESS = expectations.SUCCESS
+local FAILED = expectations.FAILED
+
+local module = {}
+local mt = { __index = { } }
+
+mt.__index.cor_id_func_map = { }
```

Comment:
@VVeremjova i have seen this table in Mobile session. Please remove it from mobiel session